torch/_dynamo/decorators.py

```diff
@@ -1002,3 +1003,10 @@ def error_on_graph_break(
     The default value of torch.compile's `error_on_graph_break` setting is False.
     """
     return ErrorOnGraphBreakDecoratorContextManager(error_on_graph_break)
+
+
+def is_dynamo_disable_recursive(method: Callable[[Any], Any]) -> Optional[bool]:
+    """
+    Check if a method is marked as `dynamo_disable` recursively.
```

Comment:
Document what should be returned:
- True if disable(recursive=True)
- False if disable(recursive=False)
- None if method is not a disable decorator